packages/cozy-client/examples/albums-relationships.js

```diff
@@ -1,4 +1,8 @@
-const { QueryDefinition, HasMany, default: CozyClient } = require('../dist')
+const {
+  QueryDefinition,
+  HasMany,
+  createClientInteractive
+} = require('cozy-client')
```

Comment:
Mmh, I think the dist was useful while developing so that you do not have to reinstall the package ? How does cozy-client is resolved since we are in the package ? Is there a symlink from cozy-client/node_modules/cozy-client to cozy-client/packages/cozy-client/dist ?

Reply:
Yes there is a simlink. With the dist import I had a syntax error error coming from cozy-ui on `import React from 'react';` I'm interested if you have a better workaround

Reply:
Mmh this is interesting, can you log an issue ? I think the primary reason is that the `import` syntax is not supported by node. You could for example use babel-node to run the example. Another solution would be for us to ship a version with `import` and a version with `requires` so that it can run either on node or via webpack. For information we do not ship versions with `require` since we typically bundle our libs through webpack and having imports is important to beneficiate from tree shaking.

https://webpack.js.org/guides/tree-shaking/

Another thing that is interesting is that cozy-clients depend on cozy-ui. I think this should not be the case, cozy-ui should not be in the import chain. I think this is because of useClientErrors ? I am not sure that its place should be cozy-client, do you think that we could move it to cozy-ui @Crash-- ?

Reply:
Moving it to UI, I'm not sure. Having it as a cozy-client plugin strongly agree!

Reply:
https://github.com/cozy/cozy-client/issues/839